src/System.Security.Cryptography.Csp/tests/CspParametersTests.cs

```diff
@@ -46,8 +46,7 @@ public static void KeyPassword_SetGet()
             }
         }
 
-        //Manual test - requires Smart Card - read instructions
-        //[Theory]
+        [Theory(Skip = "Manual test - requires Smart Card - read instructions")]
```

Comment:
Ditto on env var or something similar.
cc: @bartonjs

Reply:
Presumably `container` could get its value from an env var (TEST_CSP_SMARTCARD_CONTAINER?) and then a condition check if there's a value for that container.

But, since this is a very manual test (it'll pop UI), I'm good with Fact(Skip)